### PR TITLE
Check first for display name in Info.plist

### PR DESCRIPTION
**Pull request by irace**
https://github.com/tumblr/XExtensionItem/issues/23

### CODE REVIEW

File: Tests/XExtensionItemParametersTests.m [hunk 1]
```diff
@@ -75,6 +75,16 @@ - (void)testSourceApplication {
     XCTAssertEqualObjects(inputParams.sourceApplication, outputParams.sourceApplication);
 }
 
+- (void)testSourceApplicationFromBundle {
+    XExtensionItemMutableParameters *inputParams = [[XExtensionItemMutableParameters alloc] init];
+    inputParams.sourceApplication = [[XExtensionItemSourceApplication alloc] initWithAppNameFromBundle:[NSBundle bundleForClass:[self class]]
```
Comment (mattbischoff) [inline Tests/XExtensionItemParametersTests.m]: Wouldn’t we want the main bundle in this case?

File: XExtensionItem/XExtensionItemSourceApplication.m [hunk 2]
```diff
@@ -9,7 +9,9 @@ @implementation XExtensionItemSourceApplication
 #pragma mark - Initialization
 
 - (instancetype)initWithAppNameFromBundle:(NSBundle *)bundle appStoreID:(NSNumber *)appStoreID {
-    return [self initWithAppName:bundle.infoDictionary[(NSString *)kCFBundleNameKey] appStoreID:appStoreID];
+    NSString *displayName = bundle.infoDictionary[@"CFBundleDisplayName"] ?: bundle.infoDictionary[(NSString *)kCFBundleNameKey];
```
Comment (mattbischoff) [inline XExtensionItem/XExtensionItemSourceApplication.m]: Maybe worth making it a constant ourselves just for future readers of the code, so they know we’re not just making it up. Idk.

File: XExtensionItem.xcodeproj/xcuserdata/bryan.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist [hunk 3]
```diff
@@ -13,5 +13,21 @@
             stopOnStyle = "0">
          </BreakpointContent>
       </BreakpointProxy>
+      <BreakpointProxy
```
Comment (mattbischoff) [inline XExtensionItem.xcodeproj/xcuserdata/bryan.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist]: Why are you're breakpoints checked in?

File: Tests/XExtensionItemParametersTests.m (hunk 1 above)
Comment (irace) [inline Tests/XExtensionItemParametersTests.m]: Is the main bundle accessible from a test target?

File: XExtensionItem/XExtensionItemSourceApplication.m (hunk 2 above)
Comment (irace) [inline XExtensionItem/XExtensionItemSourceApplication.m]: Yeah, why not.

File: XExtensionItem.xcodeproj/xcuserdata/bryan.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist (hunk 3 above)
Comment (irace) [inline XExtensionItem.xcodeproj/xcuserdata/bryan.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist]: Great question!

File: Tests/XExtensionItemParametersTests.m (hunk 1 above)
Comment (mattbischoff) [inline Tests/XExtensionItemParametersTests.m]: Yes, if this is set up the way I think it is.

Comment (irace) [inline Tests/XExtensionItemParametersTests.m]: `[NSBundle mainBundle].infoDictionary` is empty, whereas `[NSBundle bundleForClass:[self class]].infoDictionary` returns the Test target’s `Info.plist` values.

Comment (mattbischoff) [inline Tests/XExtensionItemParametersTests.m]: Ok, I’m wrong. Very wrong.

**Pull request merged**